monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -478,3 +478,24 @@ def generate(self, output_folder=".", num_fold: int = 5):
                 gen_algo.export_to_disk(output_folder, name, fold=f_id)
                 algo_to_pickle(gen_algo, template_path=algo.template_path)
                 self.history.append({name: gen_algo})  # track the previous, may create a persistent history
+
+    def generate_with_customized_param(self, output_folder=".", num_fold: int = 5, **kwargs):
+        """
+        Automatically customize parameters of the bundle scripts/configs for each bundleAlgo based on gpus
```

Comment:
In the doc-string, I would explain the customization is done by running the trainings with dummy variables